Automattic/wp-calypso
# State: Modularize blog stickers state

#### Changes proposed in this Pull Request

Currently the `sites` state tree is one of the few remaining pieces of the monolithic reducer that is always loaded with Calypso. Some of its subtrees, though, seem to be unnecessarily part of it and can be further modularized.

This PR modularizes the blog stickers tree.

It also uses the opportunity to:

* Refactor `lodash` usage to native JS
* Refactor `chai` tests to use `jest`.

Reviewing commit by commit could make reviewing easier since my commits are atomic.

#### Testing instructions

* Go to `/read/a8c`.
* From the "Automattic" item in the sidebar, click on a site.
* In the feed page, right before the site title, there's an (i) icon.
* Click on that icon.
* Verify that blog stickers are retrieved and displayed properly.
* Verify that tests still pass: `yarn run test-client blog-stickers`
